### PR TITLE
WebOTP API | update details

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

update for `CredentialsContainer.get()` method `otp` option

`OTPCredential` interface, remove inherited members & add {{ReadOnlyInline}} tag and read-only word

some section title adjust due to the spec

### Additional details

see https://developer.mozilla.org/en-US/docs/Web/API/WebOTP_API and https://wicg.github.io/web-otp/

### CODE REVIEW

**Review by bsmth — APPROVED** [review]
Thanks a lot! One comment to consider, but leaving a +1 👍🏻

File: files/en-us/web/api/webotp_api/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The verification is done via a two-step process:
 1. The app client requests a one-time password (OTP), which is obtained from a specially-formatted SMS message sent by the app server.
 2. JavaScript is used to enter the OTP into a validation form on the app client and it is submitted back to the server to verify that it matches what was originally sent in the SMS.
 
-## WebOTP concepts and usage
+## Concepts and usage
```
[the remark this answers is not in the document]
Comment (skyclouds2001) [reply files/en-us/web/api/webotp_api/index.md:17]: I think the title "WebOTP API" is enough for searching

File: files/en-us/web/api/credentialscontainer/get/index.md [hunk 2]
```diff
@@ -256,6 +256,8 @@ A {{jsxref("Promise")}} that resolves with an {{domxref("OTPCredential")}} objec
 
 - `AbortError` {{domxref("DOMException")}}
   - : The `get()` operation is associated with an {{domxref("AbortSignal")}} (i.e., set in the `signal` property) that has already been aborted.
+- `NotAllowedError` {{domxref("DOMException")}}
+  - : The calling origin is an opaque origin.
```
Comment (bsmth) [inline files/en-us/web/api/credentialscontainer/get/index.md:260]: Do you want to link to this?

```suggestion
  - : The calling origin is an [opaque origin](en-US/docs/Web/HTTP/Headers/Origin#null).
```

**Review by bsmth — APPROVED** [review]
I have one comment open about opaque origin to consider. I think we need a section about opaque origins somewhere per https://whatpr.org/html/6214/origin.html#concept-origin-opaque in a follow-up PR. These changes are looking good, though, thank you!

**Comment by bsmth** [comment]
Committed the linter suggestion and going to merge shortly! Thank you!

**Pull request merged**